.gitignore

```diff
@@ -9,3 +9,11 @@
 
 # rspec failure tracking
 .rspec_status
+
+# rails support app files
```

Comment:
Moved this over from the .gitignore file originally generated in the rails support app.